js/actions/windowActions.js

```diff
@@ -738,6 +738,17 @@ const windowActions = {
   },
 
   /**
+   * Sets the fullscreen state of the window
+   * @param {boolean} isFullScreen - true if window is fullscreen
+   */
+  setFullscreenState: function (isFullScreen) {
```

Comment:
I think we need to do something to better differentiate webview and window fullscreen. The webview fullscreen state is triggered by the electron `enter-html-full-screen` event and windowActions.setFullScreen updates the `isFullScreen` property of the frame. Maybe we could have `setWebviewFullScreen' and `setWindowFullScreen` to make it more clear?